Alignment/HIPAlignmentAlgorithm/python/ALCARECOTkAlZMuMuTrackSelection_cff_py.txt

```diff
@@ -0,0 +1,159 @@
+#                                           cut = cms.string("!isFake && ndof > 4 && abs(z) <= 15 && position.Rho <= 2"), # tracksSize() > 3 for the older cut
+#                                           filter = cms.bool(True),   # otherwise it won't filter the events, just produce an empty vertex collection.
+#                                           )
+
+
+
+process.load("RecoTracker.TrackProducer.TwoBodyDecayMomConstraintProducer_cff")
+process.TwoBodyDecayMomConstraint.src = "AlignmentTrackSelector"
+process.TwoBodyDecayMomConstraint.primaryMass = cms.double(91.1876)
+process.TwoBodyDecayMomConstraint.primaryWidth = cms.double(1.4)
+#process.TwoBodyDecayMomConstraint.primaryWidth = cms.double(2.4952)
+#process.TwoBodyDecayMomConstraint.sigmaPositionCut = cms.double(0.07)
+process.TwoBodyDecayMomConstraint.rescaleError = cms.double(1.0)
+process.TwoBodyDecayMomConstraint.chi2Cut = cms.double(99999.)
+#process.TwoBodyDecayMomConstraint.EstimatorParameters.RobustificationConstant = cms.double(1.0)
+process.load("RecoTracker.TrackProducer.TwoBodyDecayConstraintProducer_cff")
+process.TwoBodyDecayConstraint.src = "AlignmentTrackSelector"
+process.TwoBodyDecayConstraint.primaryMass = cms.double(91.1876)
+process.TwoBodyDecayConstraint.primaryWidth = cms.double(1.4)
+#process.TwoBodyDecayConstraint.primaryWidth = cms.double(2.4952)
+#process.TwoBodyDecayConstraint.sigmaPositionCut = cms.double(0.07)
+process.TwoBodyDecayConstraint.rescaleError = cms.double(1.0)
+process.TwoBodyDecayConstraint.chi2Cut = cms.double(99999.)
+#process.TwoBodyDecayConstraint.EstimatorParameters.RobustificationConstant = cms.double(1.0)
+
+from TrackingTools.TrackFitters.KFFittingSmoother_cfi import KFFittingSmoother
+process.TwoBodyDecayTrackFitter = KFFittingSmoother.clone(
+    ComponentName = cms.string( "TwoBodyDecayTrackFitter" ),
+    Fitter = cms.string('RKFitter'),
+    Smoother = cms.string('RKSmoother'),
+    LogPixelProbabilityCut = cms.double(-15.0),
+    EstimateCut = cms.double(-1.0),
+)
+
+### Refitter using the regular MinBias config
+process.TrackRefitter2 = process.TrackRefitter1.clone(
+    src = "AlignmentTrackSelector"
+)
+### Refitter using the TwoBodyDecay momentum constraint
+process.TrackRefitter3 = process.TrackRefitter1.clone(
+    src = "AlignmentTrackSelector",
+    srcConstr = "TwoBodyDecayMomConstraint",
+    Fitter = cms.string('TwoBodyDecayTrackFitter'),
+    constraint = "momentum",
+    TrajectoryInEvent = True,
+    TTRHBuilder = "WithAngleAndTemplate",
+    NavigationSchool = ""
+)
+### Refitter using the full TwoBodyDecay constraint
+process.TrackRefitter4 = process.TrackRefitter1.clone(
+    src = "AlignmentTrackSelector",
+    srcConstr = "TwoBodyDecayConstraint",
+    Fitter = cms.string('TwoBodyDecayTrackFitter'),
+    constraint = "trackParameters",
+    TrajectoryInEvent = True,
+    TTRHBuilder = "WithAngleAndTemplate",
+    NavigationSchool = ""
+)
```

Comment:
Yes, but right now it is beyond the scope of this PR. As I mentioned during the TkAl meetings, switching to this sequence would probably mean re-organizing how configurations are handled.  I hope this is not a condition to merge the PR since I don't think I can find time to do this immediately.